### ISSUE TITLE
将角度转换成CSL标签是Encode部分，那有Decode部分代码吗？

### ISSUE THREAD

**Issue by tianxiu322** [hukaixuan19970627/yolov5_obb#271]
我看代码好像直接将CSL转换的tensor用于计算角度分类损失，那decode写了代码吗，还是我没看完的原因？
Encode: CSL(−Round((θgt − 90)/ω)) 
Decode: 90 − ω(Argmax(Sigmoid(logits)) + 0.5)

**Comment by hukaixuan19970627** [comment]
![image](https://user-images.githubusercontent.com/72599120/162372043-2d9ef9b9-74e7-4cd2-bbb8-1fae07efa738.png)

**Comment by tianxiu322** [comment]
> ![image](https://user-images.githubusercontent.com/72599120/162372043-2d9ef9b9-74e7-4cd2-bbb8-1fae07efa738.png)

ok，多谢
